com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -171,6 +170,7 @@
     public static final String ERROR_DURING_CLEAN_UP_0 = "Error during clean-up: {0}";
     public static final String COULD_NOT_DELETE_HISTORIC_PROCESS_0 = "Could not delete historic process \"{0}\"";
     public static final String COULD_NOT_ABORT_OPERATION_0 = "Could not abort operation \"{0}\"";
+    public static final String SKIP_SERVICES_DELETION = "Option '--delete-services' is not used. Skip deletion of services: \"{0}\".";

```

Comment:
Use "Skipping deletion of services, because the command line option \"--delete-services\" is not specified.".